napalm_nxos_ssh/nxos_ssh.py

```diff
@@ -672,11 +675,105 @@ def get_interfaces_ip(self):
         output += self.device.send_command(ipv6_command) # noqa
         return interfaces_ip
 
-    def __get_mac_address_table(self):
-        mac_table = []
+    def get_mac_address_table(self):
+        """
+        Returns a lists of dictionaries. Each dictionary represents an entry in the MAC Address
+        Table, having the following keys
+            * mac (string)
+            * interface (string)
+            * vlan (int)
+            * active (boolean)
+            * static (boolean)
+            * moves (int)
+            * last_move (float)
+        Format1:
+
+        Legend:
+        * - primary entry, G - Gateway MAC, (R) - Routed MAC, O - Overlay MAC
+        age - seconds since last seen,+ - primary entry using vPC Peer-Link,
+        (T) - True, (F) - False
+           VLAN     MAC Address      Type      age     Secure NTFY Ports/SWID.SSID.LID
+        ---------+-----------------+--------+---------+------+----+------------------
+        * 27       0026.f064.0000    dynamic      -       F    F    po1
+        * 27       001b.54c2.2644    dynamic      -       F    F    po1
+        * 27       0000.0c9f.f2bc    dynamic      -       F    F    po1
+        * 27       0026.980a.df44    dynamic      -       F    F    po1
+        * 16       0050.56bb.0164    dynamic      -       F    F    po2
+        * 13       90e2.ba5a.9f30    dynamic      -       F    F    eth1/2
+        * 13       90e2.ba4b.fc78    dynamic      -       F    F    eth1/1
+        """
+
+        #  The '*' is stripped out later
+        RE_MACTABLE_FORMAT1 = r"^\s+{}\s+{}\s+\S+\s+\S+\s+\S+\s+\S+\s+\S+".format(
```

Comment:
Yeah, my bad - it's in my nature to try to optimise things and make them run fast :-|